bridge/server/services/data-service.ts

```diff
@@ -236,6 +238,35 @@ export class DataService {
     return tasks;
   }
 
+  public async getResourcesForProjectAndService(projectName: string, serviceName: string): Promise<ServiceResource[]> {
+    const projectRes = await this.apiService.getProject(projectName);
+    const stages = projectRes.data.stages;
+
+    const resources: ServiceResource[] = [];
+
+    for (const stage of stages) {
+      let currentPage = 0;
+      let nextPage: string;
+      let resourceResponses: Resource[] = [];
+
+      do {
+        const resourceRes = await this.apiService.getServiceResource(projectName, stage.stageName, serviceName, currentPage);
+        nextPage = resourceRes.data.nextPageKey;
+        resourceResponses = [...resourceResponses, ...resourceRes.data.resources];
+
+        for (const res of resourceResponses) {
+          const serviceRes: ServiceResource = {...res, stageName: stage.stageName};
+          resources.push(serviceRes);
+        }
+
+        currentPage++;
+        // tslint:disable-next-line:radix
+      } while ((currentPage - 1) !== parseInt(nextPage));
+    }
+
+    return resources;
```

Comment:
Resources are wrong concatenated if I see correctly. Either the `for loop` inside of the `do while` should be moved outside or the `resourceResponses` should not be concatenated